README.md

```diff
@@ -63,14 +63,30 @@ yarn android
 
 The app should now open in a connected device or a running emulator and fetch the JavaScript code from the running packager.
 
-To compile and run the iOS variant of the app, use:
+To compile and run the iOS variant of the app using the _default_ simulator device, use:
 
 ```
 yarn ios
 ```
 
 which will attempt to open your app in the iOS Simulator if you're on a Mac and have it installed.
 
+### Running on Other iOS Devices
```

Comment:
Nit: Devices sound like the real hardware. Maybe we could say "Other iOS Device Simulators"?

Reply:
Removed – good catch!